### PR TITLE
added create project option in my circuits

### PR DESCRIPTION
**Pull request by yash15601**
Fixes 
https://github.com/CircuitVerse/CircuitVerse/issues/710

i have added create project option in my circuits you can see screenshot below
### Screenshots of the changes (If any) -
Screenshot (23)

### CODE REVIEW

File: config/database.example.yml [hunk 1]
```diff
@@ -1,32 +0,0 @@
-# SQLite version 3.x
```
Comment (thiefie) [inline config/database.example.yml]: This file should not be deleted(or renamed ), Please remove this from this PR

**Comment by thiefie** [comment]
Apart from that , it looks good , just fix the errors and that should be it.
Tha

File: config/database.example.yml (hunk 1 above)
Comment (MrBartusek) [reply config/database.example.yml]: and that's reason why test fail too
image

**Comment by yash15601** [comment]
All done

**Comment by Ananthavijay** [comment]
Good work @yash15601

**Comment by yash15601** [comment]
I will move this PR to another one

**Comment by yash15601** [comment]
I have moved the PR to [here](https://github.com/CircuitVerse/CircuitVerse/pull/887)

**Pull request closed without merging**